assets/Scripts/Sound/SoundManager.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+public class SoundManager : MonoBehaviour
+{
+    private static SoundManager instance;
```

Comment:
please avoid using instances for MonoBehaviours, the game manager is most of the times the only one who has a static instance. You can make this a serializablefield variable in the GameManager class, and if you want to acces the soundmanager you can do it by GameManager.Instance.SoundManager